index.bs

```diff
@@ -836,9 +836,10 @@ in a [=/Realm=] |realm|, perform the following steps:
          and |stream|'s [=relevant settings object=].
       1. If |permissionStatus| is not {{PermissionState/"granted"}},
          reject |closeResult| with a {{NotAllowedError}} and abort.
-      1. TODO: optional UA defined security checks
+      1. Perform any [=malware scans and safe browsing checks=] the user agent might want to do.
```

Comment:
nit: any [=malware scans and safe browsing checks=] the user agent might want to do -> user agent-specific [=malware scans and safe browsing checks=]?

(Assuming this doesn't lose precision, I think it's a bit shorter.)